assets/dev/js/editor/regions/panel/footer.js

```diff
@@ -119,7 +119,7 @@ module.exports = Marionette.ItemView.extend( {
 			// Force exit if device mode is already desktop
 			elementor.exitDeviceMode();
 		} else {
-			const deviceView = 'default' === elementor.getPreferences( 'default_device_view' ) ? 'desktop' : elementor.getPreferences( 'default_device_view' );
+			const deviceView = 'desktop';
```

Comment:
you can remove the whole `else` I  think
and probably this whole footer file

Reply:
Deleted in a separate PR.